pkg/datarepair/repairer/repairer.go

```diff
@@ -11,30 +11,27 @@ import (
 
 	"storj.io/storj/internal/sync2"
 	"storj.io/storj/pkg/datarepair/queue"
-	"storj.io/storj/pkg/pb"
-	segment "storj.io/storj/pkg/storage/segments"
+	"storj.io/storj/pkg/storage/segments"
 	"storj.io/storj/storage"
 )
 
 // Repairer is the interface for the data repairer
 type Repairer interface {
-	//do we need this method? It doesn't look implemented
-	Repair(ctx context.Context, seg *pb.InjuredSegment) error
 	Run(ctx context.Context) error
 }
 
 // repairer holds important values for data repair
 type repairer struct {
 	queue   queue.RepairQueue
-	store   segment.Store
+	sr      segments.Repairer
```

Comment:
`repairer`

Reply:
Not really, but it sounds better than `sr`.

Reply:
OK. Done. I also renamed the existing `repairer` to `service` to avoid the name collision.